src/renderer/i18n/en.js

```diff
@@ -165,7 +165,8 @@ const English = {
     connect: "Connect",
     disconnect: "Disconnect",
     scan: "Scan keyboards",
-    permissionError: `Insufficient permissions, please make sure the device file is read- and writeable!`
+    installUdevRules: "Install Udev rules",
+    permissionError: `Cannot access the device file. Please make sure it is read- and writeable!`
```

Comment:
Your computer won't let Chrysalis talk to your keyboard. (You do not have read/write permissions to %1.)

Chrysalis can fix this by installing a udev rules file into  /etc/udev/rules.d/

---
%1 should be the device file.

Reply:
Tangentially, - the centered red text on white background pattern looks...not great. does material design let us use some kind of error box?

Reply:
One way would be to use a different background, as shown [here](https://www.material.io/design/color/text-legibility.html#text-backgrounds) - but for that to work, I'd have to move the error box and the button to below the appbar, instead of below the key selector, because a colored box in the middle of the screen would look terrible.

As far as I see, the idiomatic way to display errors is either a snackbar (not suitable in this case), or as a bar below the appbar. Reading [further](https://www.material.io/design/communication/confirmation-acknowledgement.html#acknowledgement), the guidelines say that we should be using alerts here. While the example there doesn't use a different background, I think _we_ should, to bring attention to the problem.

We could name the button "Fix it" too, then. I like that better than "Fix permission errors".

Reply:
Sure, though if it's more than 10 minutes work, I'd rather it be a follow-up PR.

Reply:
It's... ~3 minutes of work. Except it's not, because of dependencies. The Material UI lib we use has a very nice alert component, but only available on its 4.x branch (we're currently on 3.x). Upgrading to 4.x is flawless, there's only a single change that needs to be done, because between 3.x and 4.x, one component graduated from Labs to Core. Yay! We have pretty alerts:

![Screenshot from 2020-10-07 20-45-46](https://user-images.githubusercontent.com/17243/95374012-17cced00-08de-11eb-8528-2b49c66dfbd3.png)

But upgrading broke our notifications. The notistack package had a fix committed yesterday, and a new release was promised, but it's not available yet. So until the fix is out, we either have nice alerts and broken notifications, or working notifications and meh alerts.

At the moment, I'm leaning towards keeping this as a draft until notistack is fixed, or if that doesn't happen in the next day or two, I'll roll back the alert changes and go with the rest.